src/main/java/com/stripe/model/issuing/Dispute.java

```diff
@@ -89,6 +89,13 @@ public class Dispute extends ApiResource
   @SerializedName("status")
   String status;
 
+  /**
+   * The enum that describes the dispute loss outcome. If the dispute is not lost, this field will
+   * be absent.
+   */
+  @SerializedName("loss_reason")
```

Comment:
nit: For reference, might be nice to point to the doc where this field is described, since not present in the current Stripe model.